dotnet/src/SemanticKernel/KernelExtensions/ImportSemanticSkillFromDirectory.cs

```diff
@@ -53,48 +53,50 @@ public static class ImportSemanticSkillFromDirectoryExtension
     /// </summary>
     /// <param name="kernel">Semantic Kernel instance</param>
     /// <param name="parentDirectory">Directory containing the skill directory, e.g. "d:\myAppSkills"</param>
-    /// <param name="skillDirectoryName">Name of the directory containing the selected skill, e.g. "StrategySkill"</param>
+    /// <param name="skillDirectoryNames">Name of the directories containing the selected skills, e.g. "StrategySkill"</param>
     /// <returns>A list of all the semantic functions found in the directory, indexed by function name.</returns>
     public static IDictionary<string, ISKFunction> ImportSemanticSkillFromDirectory(
-        this IKernel kernel, string parentDirectory, string skillDirectoryName)
+        this IKernel kernel, string parentDirectory, params string[] skillDirectoryNames)
     {
         const string CONFIG_FILE = "config.json";
         const string PROMPT_FILE = "skprompt.txt";
 
-        Verify.ValidSkillName(skillDirectoryName);
-        var skillDir = Path.Combine(parentDirectory, skillDirectoryName);
-        Verify.DirectoryExists(skillDir);
-
         var skill = new Dictionary<string, ISKFunction>();
 
-        string[] directories = Directory.GetDirectories(skillDir);
-        foreach (string dir in directories)
+        foreach (string skillDirectoryName in skillDirectoryNames)
```

Comment:
nit: wondering if it's worth doing this in parallel.

Reply:
Especially after #603

Reply:
I'm good on merging this one and working on it later as Shawn suggested.